mobile/library/cc/engine_builder.h

```diff
@@ -75,7 +75,7 @@ class EngineBuilder {
   EngineBuilder& addDnsPreresolveHostnames(const std::vector<std::string>& hostnames);
   EngineBuilder& addNativeFilter(std::string name, std::string typed_config);
   EngineBuilder& enableAdminInterface(bool admin_interface_on);
-  EngineBuilder& addStatsSink(std::string name, std::string typed_config);
+  EngineBuilder& addStatsSinks(const std::vector<std::string>& stat_sinks);
```

Comment:
(if we change the signature in the .cc file, we need to change the signature here, of course)